sirepo/template/radia.py

```diff
@@ -394,13 +420,18 @@ def _get_sdds():
     return _cfg.sdds
 
 
-def _read_data(sim_id, view_type, field_type):
+def _read_path(sim_id, h5path):
     try:
         with h5py.File(_geom_file(sim_id), 'r') as hf:
-            g = template_common.h5_to_dict(hf, path=_geom_h5_path(view_type, field_type))
-        return g
+            return template_common.h5_to_dict(hf, path=h5path)
     except IOError:
```

Comment:
This is too coarse of a test. If the file is "not found", then we should know that. There's a call for this in pkio, which you should use so it is compatible with py.path and future not founds.